sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -274,6 +274,78 @@ case class AlterTableUnsetPropertiesCommand(
 
 }
 
+
+/**
+ * A command to change the columns for a table, only support changing the comments of non-partition
+ * columns for now.
+ *
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   ALTER TABLE table_identifier
+ *   CHANGE [COLUMN] column_old_name column_new_name column_dataType [COMMENT column_comment]
+ *   [FIRST | AFTER column_name];
```

Comment:
This is the right Hive syntax, but `SparkSqlParser.scala` is using the different one.